### ISSUE TITLE
Can't specify animation-duration using inline css

### ISSUE THREAD

**Issue by BenLorantfy** [NativeScript/NativeScript#6225]
### Tell us about the problem
Specifying animation-duration with inline CSS doesn't have an effect.

### Which platform(s) does your issue occur on?
- Only tested on iOS

### Please provide the following version numbers that your issue occurs with:
- CLI:  4.1.2
- Cross-platform modules: 4.2.0
- Runtime(s): 4.1.0

### Please tell us how to recreate the issue in as much detail as possible.
- create an animation in a stylesheet with a class
- add that class to a component
- try over-riding the animation-duration using inline css

See: https://play.nativescript.org/?template=play-ng&id=7vCE99

**Comment by NickIliev** [comment]
@BenLorantfy indeed the CSS animations are supported via CSS files and not via XML properties. You could also use code-behind to access and modify [your CSS animation](https://docs.nativescript.org/ui/animation-css#access-css-animations-from-code).

**Comment by BenLorantfy** [comment]
@NickIliev I'll probably end up using that solution, but I'd like to keep my animations declarative if possible.

**Comment by NickIliev** [comment]
@BenLorantfy I think that changing the animation speed is not possible via the declarative way (discussed [here](https://stackoverflow.com/a/27558476/4936697)) so I guess going through the JavaScript is the way to go (e.g. with changing classes)

**Comment by BenLorantfy** [comment]
@NickIliev I don't see why this should be closed. Is it not expected behaviour that all css properties should be supported using inline css? Why should there be a difference between a stylesheet and inline-css ?

**Comment by NickIliev** [comment]
@BenLorantfy you are right - reopening the issue as an enhancement (we will discuss if this is related to a bug in the animations).
